src/styles/components/charts/styles.less

```diff
@@ -1,342 +1,580 @@
-        }
+      &.path-color-3 {
+        stroke: @blue;
+      }
 
-        path {
-          stroke: @neutral;
-        }
+      &.path-color-4 {
+        stroke: @green;
+      }
+
+      &.path-color-5 {
+        stroke: @orchid;
       }
 
-      line {
-        display: none;
+      &.path-color-6 {
+        stroke: @pink;
       }
+
     }
 
-    .grid-graph {
-      .grid.y {
-        .tick:first-child line {
-          stroke-dasharray: 0,0;
+    .barchart {
+
+      .bar {
+        stroke-width: 0px;
+
+        line {
+          fill-opacity: 1;
+          stroke-width: 2px;
+
+          &.hidden,
+          &.invisble {
+            stroke-width: 0px;
+          }
+
         }
+
       }
 
-      &.inverse line {
-        stroke: fade(@neutral, 10%);
+      .axis {
+
+        &.x path {
+          display: none;
+        }
+
+        &.y {
+
+          &.inverse path {
+            stroke: fade(@neutral, 10%);
+          }
+
+          path {
+            stroke: @neutral;
+          }
+
+        }
+
+        line {
+          display: none;
+        }
+
       }
 
-      line {
-        stroke: @neutral;
-        stroke-dasharray: 4,4;
+      .grid-graph {
+
+        .grid.y {
+
+          .tick:first-child line {
+            stroke-dasharray: 0,0;
+          }
+
+        }
+
+        &.inverse line {
+          stroke: fade(@neutral, 10%);
+        }
+
+        line {
+          stroke: @neutral;
+          stroke-dasharray: 4,4;
+        }
+
       }
-    }
-  }
 
-  .timeseries-chart {
-    position: absolute;
-    left: 0;
-    top: 0;
+    }
 
-    .moving-elements {
-      position: absolute;
+    .timeseries-chart {
       left: 0;
+      position: absolute;
       top: 0;
-      transform: translateZ(0);
+
+      .moving-elements {
+        left: 0;
+        position: absolute;
+        top: 0;
+        transform: translateZ(0);
+      }
+
     }
-  }
 
-  .timeseries-chart,
-  .barchart {
+    .timeseries-chart,
+    .barchart {
 
-    .axis {
+      .axis {
 
-      &.x,
-      &.y {
-        font-size: @small-text-font-size;
+        &.x,
+        &.y {
+          font-size: 0.8rem;
+
+          .domain {
+            display: none;
+          }
 
-        .domain {
-          display: none;
         }
+
       }
+
     }
-  }
 
-  .unit-bordered-horizontal {
-    border-right: 1px solid @border-color;
+    .unit-bordered-horizontal {
+      border-right: 1px solid @chart-axis-color;
 
-    &:last-child {
-      border-right: none;
-    }
+      &:last-child {
+        border-right: none;
+      }
 
-    .panel-inverse & {
-      border-color: @border-color-inverse;
-    }
-  }
+      .panel-inverse & {
+        border-color: @chart-axis-color-inverse;
+      }
 
-  & when (@typography-enabled) {
-    .unit {
-      fill: @white;
-      line-height: 1;
     }
 
-    .unit-label {
-      fill: @purple;
-      &.path-color-0 {
-        color: @purple;
-      }
+    & when (@typography-enabled) {
```

Comment:
What is the point of this check?  Looking at the applied styles, none seem to be dependent on Typography from CNVS.